application/controllers/HostController.php

```diff
@@ -2,21 +2,54 @@
 
 namespace Icinga\Module\Businessprocess\Controllers;
 
-use Icinga\Module\Monitoring\Controller;
+use Icinga\Module\Businessprocess\Common\IcingadbDatabase;
+use Icinga\Module\Businessprocess\Web\Controller;
+use Icinga\Module\Icingadb\Model\Host;
+use Icinga\Module\Monitoring\Backend;
 use Icinga\Web\Url;
 
 class HostController extends Controller
 {
+    use IcingadbDatabase;
+
+    protected $backend;
```

Comment:
The member `$backend` is already there. The controller inherits it.
After all, the original code already accesses `$this->backend`.

Reply:
In the original code we were using the controller from monitoring module, I have changed that to Icinga\Module\Businessprocess\Web\Controller. Here the member $backend is not present. If I use the controller from the monitoring module, it does not work for IcingaDB backend.